src/components/screen/Chat.tsx

```diff
@@ -179,13 +181,99 @@ function Screen(props: Props): React.ReactElement {
                 justifyContent: 'center',
                 alignItems: 'center',
               }}
+              onPress={(): void => Alert.alert('Image')}
             >
               <Ionicons
                 name="md-images"
                 size={36}
                 color={theme ? theme.fontColor : '#3d3d3d'}
               />
             </TouchableOpacity>
+            <TouchableOpacity
+              style={{
+                marginLeft: 16,
+                marginTop: 4,
+                width: 60,
+                height: 60,
+                justifyContent: 'center',
+                alignItems: 'center',
```

Comment:
This line can be shortened with `StyledComponent`. Line 209 has exactly the same style.